File: sasang/src/main/java/com/pluxity/domains/device/dto/DeviceUpdateRequest.java
package com.pluxity.domains.device.dto;

import com.pluxity.feature.dto.FeatureUpdateRequest;

public record DeviceUpdateRequest(
        FeatureUpdateRequest feature,
        Long deviceCategoryId,
        Long stationId,
        Long asset,
        Long iconId,
        String name,
        String code,
        String description) {}

File: common/src/main/kotlin/com/pluxity/device/dto/DeviceUpdateRequest.kt
package com.pluxity.device.dto

import com.pluxity.device.entity.DeviceCompanyType
import com.pluxity.device.entity.DeviceType

class DeviceUpdateRequest(
    val name: String,
    val deviceType: DeviceType,
    val companyType: DeviceCompanyType,
)
